wicket-metrics/src/main/resources/META-INF/aop.xml

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE aspectj PUBLIC "-//AspectJ//DTD//EN" "http://www.eclipse.org/aspectj/dtd/aspectj.dtd">
+<aspectj>
+    <weaver options="-nowarn">
+        <include within="org.apache.wicket..*"/>
+    </weaver>
+    <aspects>
+		<aspect name="org.apache.wicket.metrics.aspects.BehaviorAspect" />
+		<aspect name="org.apache.wicket.metrics.aspects.IPartialPageRequestHandlerAspect" />
+		<aspect name="org.apache.wicket.metrics.aspects.ComponentAspect" />
+		<aspect name="org.apache.wicket.metrics.aspects.WicketFilterAspect" />
```

Comment:
This answer combined with the one for my question on IPartialPageRequestHandlerAspect makes me think it would be better to not add any aspect by default.
Each application has to decide which ones to use for itself.